extensions/openshift-ai/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "lib": ["ES2017", "webworker"],
+    "sourceMap": true,
+    "rootDir": "src",
+    "outDir": "dist",
+    "skipLibCheck": true,
+    "types": ["node"]
+  },
```

Comment:
suggestion: possibility to move other extension tsconfig to a template one at the root of extensions folder
and extend them
https://www.typescriptlang.org/tsconfig/#extends